user.go

```diff
@@ -71,6 +90,25 @@ func (c *Client) UnmuteUser(targetID string, userID string) error {
 	return c.makeRequest(http.MethodPost, "moderation/unmute", nil, data, nil)
 }
 
+// Removes a mute
+// targetID: the user getting un-muted
+// userID: the user muting the target
+func (c *Client) UnmuteUsers(targetIDs []string, userID string) error {
+	switch {
+	case len(targetIDs) == 0:
+		return errors.New("target IDs is empty")
+	case userID == "":
+		return errors.New("user ID is empty")
+	}
+
+	data := map[string][]string{
+		"target_ids": targetIDs,
+		"user_id":    {userID},
```

Comment:
why are you sending `user_id` as []string? either server-side we do something weird or this client is weird

Reply:
@tbarbugli `url.Values` does have signature `map[string][]string`